packages/app/background-jobs-common/UPGRADING.md

```diff
@@ -0,0 +1,45 @@
+# Upgrading Guide
+
+## Upgrading from `4.0.0` to `5.0.0`
+
+### Description of Breaking Changes
+
+1. **Property Removal**
+  - The deprecated `processCalls` property from `FakeBackgroundJobProcessor` has been removed.
+
+2. **Method Changes in `AbstractBackgroundJobProcessor`**
+  - The method `getActiveQueueIds` has been removed. A new constant method now serves this purpose.
+  - `logJobStarted` and `logJobFinished` methods are no longer public. The SDK now offers basic logging, and there are
+    alternative ways to extend logging if needed.
+  - The method `resolveExecutionLogger` can no longer be overridden. The SDK will now automatically create the 
+    recommended logger.
+
+3. **Visibility Changes for Constants and Methods**
+  - The following constants were not designed to be public and are no longer accessible:
+    - `RETENTION_COMPLETED_JOBS_IN_AMOUNT`
+    - `RETENTION_FAILED_JOBS_IN_DAYS`
+    - `RETENTION_QUEUE_IDS_IN_DAYS`
+  - The following methods were not designed to be public and are no longer accessible:
+    - `daysToSeconds`
+    - `daysToMilliseconds`
+    - `isUnrecoverableJobError`
+    - `sanitizeRedisConfig`
+
+### Migration Steps
+
+#### For `FakeBackgroundJobProcessor`
+- If you were using the `processCalls` property, you should remove it and switch to using the spy feature which was 
```

Comment:
can you link to the corresponding documentation section?

Reply:
Done 🙌 thank youuu